lib/jelix-admin-modules/jauthdb_admin/controllers/default.classic.php

```diff
@@ -1,82 +1,89 @@
 <?php
 /**
-* @package   admin
-* @subpackage jauthdb_admin
-* @author    Laurent Jouanneau
-* @copyright 2009-2019 Laurent Jouanneau
-* @link      http://jelix.org
-* @license   http://www.gnu.org/licenses/old-licenses/gpl-2.0.html GNU Public Licence
-*/
+ * @author    Laurent Jouanneau
+ * @copyright 2009-2019 Laurent Jouanneau
+ *
+ * @see      http://jelix.org
+ *
+ * @license   http://www.gnu.org/licenses/old-licenses/gpl-2.0.html GNU Public Licence
+ */
 
 /**
- * controller to manage all users
+ * controller to manage all users.
  */
-class defaultCtrl extends jController {
-
+class defaultCtrl extends jController
+{
     public $sensitiveParameters = array('password', 'password_confirm', 'pwd', 'pwd_confirm');
 
-    public $pluginParams=array(
-        'index'        =>array('jacl2.right'=>'auth.users.list'),
-        'view'         =>array('jacl2.right'=>'auth.users.view'),
-        'precreate'    =>array('jacl2.rights.and'=>array('auth.users.view','auth.users.create')),
-        'create'       =>array('jacl2.rights.and'=>array('auth.users.view','auth.users.create')),
-        'savecreate'   =>array('jacl2.rights.and'=>array('auth.users.view','auth.users.create')),
-        'preupdate'    =>array('jacl2.rights.and'=>array('auth.users.view','auth.users.modify')),
-        'editupdate'   =>array('jacl2.rights.and'=>array('auth.users.view','auth.users.modify')),
-        'saveupdate'   =>array('jacl2.rights.and'=>array('auth.users.view','auth.users.modify')),
-        'deleteconfirm'=>array('jacl2.rights.and'=>array('auth.users.view','auth.users.delete')),
-        'delete'       =>array('jacl2.rights.and'=>array('auth.users.view','auth.users.delete')),
+    public $pluginParams = array(
+        'index' => array('jacl2.right' => 'auth.users.list'),
```

Comment:
php-cs-fixer is not good. it is less readable than the previous code. Is there a configuation parameter we can set to avoid this change?

Reply:
There is but we should do a whole PR to fix it in every file otherwise the lint will fail everytime.